lib/active_model/serializer/adapter/flatten_json.rb

```diff
@@ -1,19 +1,13 @@
-module ActiveModel
-  class Serializer
-    class Adapter
-      class FlattenJson < Json
-        def serializable_hash(options = {})
-          super
-          @result
-        end
+class ActiveModel::Serializer::Adapter::FlattenJson < ActiveModel::Serializer::Adapter::Json
```

Comment:
I find the previous style more readable. That's also the syntax (I think) Rails uses internally, perhaps it would be a good idea to align with Rails code style.